README.md

```diff
@@ -1,17 +1,32 @@
-# mcrouter
-[![Build Status](https://travis-ci.org/evertrue/mcrouter-cookbook.svg)](https://travis-ci.org/evertrue/mcrouter-cookbook)
```

Comment:
kitchen runs fine I think something changed on the way it connected to travis-ci. I think the original maintainer is trying to work it out on the master branch.